deepchem/feat/base_classes.py

```diff
@@ -207,6 +210,198 @@ def __call__(self, molecules):
     return self.featurize(molecules)
 
 
+class StructureFeaturizer(Featurizer):
```

Comment:
Could we give this a more specific name?  "Structure" can refer to many different things, not just to inorganic crystal structures.

Reply:
Sure, how about `MaterialStructureFeaturizer`? I stayed away from "Crystal" to avoid confusion with protein crystal structures, etc.

Reply:
Sounds good!